### DISCUSSION TITLE
Paprika Import

### DISCUSSION THREAD

**Discussion by Justinos08** [TandoorRecipes/recipes#2804] [Q&A]
I am trying to import my recipes from Paprika and the upload stalls at around 2 thirds the way. I have also tried to import it in the Mealie app and I run into the same issue which would indicate that Paprika is exporting corrupt data. Any ideas of any logs or steps I can take to help me find what is wrong with the Paprika export.

**Comment by smilerz** [comment]
CN you manually try to split the export into several pieces?

**Comment by Justinos08** [comment]
That was a good suggestion @smilerz. It turns out it was a recipe that was in a cookbook that had no name that was causing the issue.
